src/pages/callback-list.js

```diff
@@ -9,21 +9,23 @@ import { CallTypesGateway } from '../gateways/call-types';
 
 function CallbacksListPage({ callTypes }) {
     const [callbacks, setCallbacks] = useState([]);
+    const [subsetCallbacks, setSubsetCallbacks] = useState([]);
     const [callHandlers, setCallHandlers] = useState([]);
     const [dropdowns, setDropdowns] = useState({
         callType: 'All',
         assignedTo: 'Assigned to all'
     });
 
     const getCallBacks = async () => {
-        const queryParams = { ...dropdowns };
-        if (queryParams.callType === 'All') delete queryParams['callType'];
-        if (queryParams.assignedTo === 'Assigned to all') delete queryParams['assignedTo'];
+        // const queryParams = { ...dropdowns };
+        // if (queryParams.callType === 'All') delete queryParams['callType'];
+        // if (queryParams.assignedTo === 'Assigned to all') delete queryParams['assignedTo'];
 
         const gateway = new CallbackGateway();
-        const callbackList = await gateway.getCallback(queryParams);
+        const callbackList = await gateway.getCallback({}); //queryParams);
```

Comment:
I think maybe it'd be worth moving this into getInitialProps or getServerSideProps so that the callbacks info id passed into the page as a prop (?)

Reply:
Oh yes, about that... the current endpoint takes a filtering parameter for filtering by help needed type. However it does not support filtering by call handler. So I was left with 2 options: 
1. filter half remotely (help type), half locally (call handlers)
2. filter everything locally.

In order to make everything simpler, I rewrote the filtering so it would be done client-side, hence there isn't a need currently to pass in a proper query object into an api call. However I left the code commented out, in case the endpoint gets extended, and we for some reason decide to go back to filtering remotely (via api call) 😓

Reply:
Sure, we can move this to get initial props. Though, I would prefer to do it some other PR as this one has a lot of changes under it already 😅

Reply:
Fair enough!